zebra-chain/src/error.rs

```diff
@@ -0,0 +1,38 @@
+//! Errors that can occur inside any `zebra-chain` submodule.
+use thiserror::Error;
+
+/// Number of times a `diversify_hash` will try to obtain a diversified base point.
+pub const DIVERSIFY_HASH_TRIES: u8 = 5;
```

Comment:
Let's go with 2 total tries:

```suggestion
pub const DIVERSIFY_HASH_TRIES: u8 = 2;
```

Reply:
I would actually move this const to the sapling/keys module as that's where it's used, including the number of tries in the error message is not something a general user can change or take an action on, so we don't strictly need it in the message